core/services/request/middleware/DetectFileEditorRequest.php

```diff
@@ -28,34 +28,10 @@ public function handleRequest(RequestInterface $request, ResponseInterface $resp
     {
         $this->request  = $request;
         $this->response = $response;
-        if ($this->isFileEditorRequest()) {
-            $this->setFiltersForRequest();
+        if ($this->request->isWordPressScrape()) {
+            add_filter('FHEE_load_EE_Session', '__return_false', 999);
```

Comment:
so this is saying we don't need the session during WP scrape requests, right? Again IMO it would be good to explain why. (eg, although scrape requests use cookies, they shouldn't try to store or use the session)

Reply:
sure will do